src/exercises/exerciseTypes/multipleChoiceContext/MultipleChoiceContext.js

```diff
@@ -0,0 +1,157 @@
+        initExerciseBookmarks[i].isExercise = false
+    }
+    setExerciseBookmarks(initExerciseBookmarks);
+    api.getArticleInfo(bookmarksToStudy[0].article_id, (articleInfo) => {
+      setInteractiveText(
+        new InteractiveText(
+          bookmarksToStudy[0].context,
+          articleInfo,
+          api,
+          "TRANSLATE WORDS IN EXERCISE",
+          EXERCISE_TYPE,
+          speech,
+        ),
+      );
+    setArticleInfo(articleInfo);
+    });
+  }, []);
+
+  useEffect(() => {
+    if (articleInfo && interactiveText) {
+      setExerciseBookmarks(shuffle(
+        [...exerciseBookmarks]
+      ));
+    }
+  }, [articleInfo, interactiveText, bookmarksToStudy]);
+
+  function handleShowSolution() {
+    let message = messageToAPI + "S";
+    notifyIncorrectAnswer(bookmarksToStudy[0]);
+    setIsCorrect(true);
+    handleAnswer(message);
+    setShowSolution(true);
+  }
+
+ function notifyChoiceSelection(selectedChoice, index) {
+    if(isCorrect) return;
+    setClickedOption(index);
+    if (
+        selectedChoice === bookmarksToStudy[0].context
+    ) {
+        setShowSolution(true);
+        setClickedIndex(index);
+        correctAnswer(bookmarksToStudy[0].context);
+        setIsCorrect(true);
+        let concatMessage = messageToAPI + "C";
+        handleAnswer(concatMessage);
+    } else {
+        setClickedIndex(null);
+        setIncorrectAnswer(selectedChoice);
+        notifyIncorrectAnswer(bookmarksToStudy[0].context);
+        let concatMessage = messageToAPI + "W";
+        setMessageToAPI(concatMessage);
+            setTimeout(() => {
+              setClickedOption(null);
+            }, 500)
+    }
+  }
+
+  function handleAnswer(message) {
+    setMessageToAPI(message);
+    api.uploadExerciseFinalizedData(
+      message,
+      EXERCISE_TYPE,
+      getCurrentSubSessionDuration(activeSessionDuration, "ms"),
+      bookmarksToStudy[0].id,
+      exerciseSessionId,
+    );
+  }
+
+  function getHighlightedWord(word){
+    return `<span class="highlightedWord">${word}</span>`;
+  }
+
+  if (!articleInfo || !interactiveText) {
+    return <LoadingAnimation />;
+  }
+
+  return (
+    <s.Exercise className="findWordInContext">
+      <div className="headlineWithMoreSpace">
+        {strings.multipleChoiceContextHeadline}
+      </div>
+      <h1 className="wordInContextHeadline">{bookmarksToStudy[0].from}</h1>
+        {exerciseBookmarks.map((option, index) => (
+            <s.MultipleChoiceContext 
+                key={index} 
+                clicked={index === clickedIndex}
+                isCorrect={isCorrect}
+                className={clickedOption !== null ? (index === clickedOption) ? (option.isExercise) ? "correct" : "wrong" : "" : ""}
+                onClick={(e) => notifyChoiceSelection(option.context, index, e)}>
+                <div dangerouslySetInnerHTML={{ __html: showSolution
+                        ? option.context.replace(option.from, getHighlightedWord(option.from))
```

Comment:
I think we should highlight the other options as just bold, rather than colored - what do you think?

Reply:
<img width="404" alt="Screenshot 2024-03-22 at 12 37 30" src="https://github.com/zeeguu/web/assets/147062323/eaff46e8-5447-415e-a5ff-de795b541860">
I think you are right. Having the orange border is enough.

Reply:
I think you can keep the yellow word in the solution for consistency, but it looks nicer with black for the others for sure!